inst/rmarkdown/resources/css/default-page.css

```
@@ -267,14 +330,20 @@ div.front-page h2.date {
 .section > h1, .section > h2, .section > h3, .section > h4, .section > h5, .section > h6 {
   break-after: avoid;
 }
-.footenotes {
+.footnotes {
   break-before: always;
   break-after: always;
+  color: red;
 }
 .figure {
   break-inside: avoid;
 }
 
+a[class='footnote-ref'] {
```

Plus sobre, plus sémantique et plus efficace

```suggestion
.footnote-ref {
```